tests/garage/misc/test_wrappers.py

```diff
@@ -0,0 +1,20 @@
+import gym
+
+from garage.misc.wrappers import ResizeAndGrayscaleWrapper, StackFramesWrapper
+from tests.fixtures import TfGraphTestCase
+
+
+class TestWrappers(TfGraphTestCase):
+    def test_resize_and_gray_scale_wrapper(self):
+        env = ResizeAndGrayscaleWrapper(gym.make("Breakout-v0"), w=84, h=84)
+        assert env.observation_space.shape == (84, 84)
+
+    def test_stack_frames_wrapper(self):
+        # StackFrameWrapper only supports input with channel=1
+        env = StackFramesWrapper(
+            ResizeAndGrayscaleWrapper(gym.make("Breakout-v0"), w=84, h=84),
+            n_frames_stacked=4)
+
+        env.reset()
+        obs, _, _, _ = env.step(0)
+        assert obs.shape == (84, 84, 4)
```

Comment:
You can verify a lot more about this than its size, so you should. You know exactly what the contents should contain.